tests/Unit/DataStructures/DataBox/Test_DataBoxDocumentation.cpp

```diff
@@ -0,0 +1,512 @@
+}
+
+/// [tagged_tuple_mass_compute]
+template<typename... Tags>
+double mass_compute(const tuples::TaggedTuple<Tags...>& box) noexcept {
+
+  return tuples::get<Density2>(box) * tuples::get<Volume2>(box);
+}
+/// [tagged_tuple_mass_compute]
+
+/// [tagged_tuple_acceleration_compute]
+template<typename... Tags>
+double acceleration_compute(const tuples::TaggedTuple<Tags...>& box) noexcept {
+
+  return tuples::get<Velocity2>(box) * tuples::get<Velocity2>(box) /
+         tuples::get<Radius2>(box);
+}
+/// [tagged_tuple_acceleration_compute]
+
+/// [tagged_tuple_force_compute]
+template<typename... Tags>
+double force_compute(const tuples::TaggedTuple<Tags...>& box) noexcept {
+
+  const double mass = mass_compute(box);
+  const double acceleration = acceleration_compute(box);
+  return mass * acceleration;
+}
+/// [tagged_tuple_force_compute]
+
+/// [proper_databox_tags]
+struct Velocity3 : db::SimpleTag {
+  using type = double;
+  static std::string name() noexcept { return "Velocity3"; }
+};
+struct Radius3 : db::SimpleTag {
+  using type = double;
+  static std::string name() noexcept { return "Radius3"; }
+};
+struct Density3 : db::SimpleTag {
+  using type = double;
+  static std::string name() noexcept { return "Density3"; }
+};
+struct Volume3 : db::SimpleTag {
+  using type = double;
+  static std::string name() noexcept { return "Volume3"; }
+};
+struct Mass : db::SimpleTag {
+  using type = double;
+  static std::string name() noexcept { return "Mass"; }
+};
+/// [proper_databox_tags]
+
+double refined_databox_1(){
+/// [refined_databox]
+const auto refined_databox = db::create<
+    db::AddSimpleTags<
+      Velocity3, Radius3, Density3, Volume3>>(4.0, 2.0, 0.5, 10.0);
+/// [refined_databox]
+/// [refined_databox_get]
+const double velocity = db::get<Velocity3>(refined_databox);
+/// [refined_databox_get]
+const double radius = db::get<Radius3>(refined_databox);
+const double density = db::get<Density3>(refined_databox);
+const double volume = db::get<Volume3>(refined_databox);
+return density * volume *  velocity * velocity / radius;
+}
+
+/// [compute_tags]
+struct MassCompute : db::ComputeTag, Mass {
+  static std::string name() noexcept { return "MassCompute"; }
+  static constexpr auto function(const double& density, const double& volume){
+    return density * volume; }
+  using argument_tags = tmpl::list<Density3, Volume3>;
+};
+/// [compute_tags]
+
+/// [compute_tags_force_compute]
+struct Force : db::SimpleTag {
+  using type = double;
+  static std::string name() noexcept { return "Force"; }
+};
+
+struct ForceCompute : db::ComputeTag, Force {
+  static std::string name() noexcept { return "ForceCompute"; }
+  static constexpr auto function(
+    const double& velocity, const double& radius,
+    const double& density, const double& volume){
+    //Compute mass * acceleration:
+    return density * volume * velocity * velocity / radius; }
+  using argument_tags = tmpl::list<Velocity3, Radius3, Density3, Volume3>;
```

Comment:
grab `mass` and `acceleration` from the DataBox (add the `acceleration` compute tag)